### PR TITLE
Add L2C capability monitor

### PR DESCRIPTION
**Pull request by rge-exafore**

### CODE REVIEW

File: src/l2c_capb.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{
+  gps_l2cm_l2c_capabilities = l2c_capb;
+}
+
+u32 gps_l2cm_l2c_cap_read()
+{
+  return gps_l2cm_l2c_capabilities;
+}
+
+
```
Comment (pmiettinen) [inline src/l2c_capb.c]: Too many empty lines.

Comment (rge-exafore) [inline src/l2c_capb.c]: Fixed.

File: src/l2c_capb.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include "l2c_capb.h"
+
+/* Initial value matches the status of GPS constellation on 2016-05-17 */
+static volatile u32 gps_l2cm_l2c_capabilities = 0xf7814bfd;
```
Comment (jacobmcnamee) [inline src/l2c_capb.c]: No need for volatile

File: src/decode/decode_gps_l1ca.c [hunk 3]
```diff
@@ -101,18 +102,25 @@ static void decoder_gps_l1ca_process(const decoder_channel_info_t *channel_info,
...
 
-  /* Decoded a new ephemeris. */
-  ephemeris_new(&e);
 
-  ephemeris_t *eph = ephemeris_get(channel_info->sid);
-  if (!eph->valid) {
-    log_info_sid(channel_info->sid, "ephemeris is invalid");
+  if (dd.gps_l2c_sv_capability_upd_flag) {
+    /* Store new L2C value  */
+    log_info("L2C capabilities received: 0x%x", dd.gps_l2c_sv_capability);
```
Comment (jacobmcnamee) [inline src/decode/decode_gps_l1ca.c]: I vote we remove this log statement since it is irrelevant and confusing for end users on v2.

**Comment by jacobmcnamee** [comment]
lgtm, just update libswiftnav pointer

**Pull request merged**